egs2/TEMPLATE/asr1/pyscripts/utils/evaluate_pseudomos.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
```

Comment:
You may consider update the README at `egs2/TEMPLATE/tts1`

Reply:
Good point, thanks! Added the readme.